airflow/executors/base_executor.py

```diff
@@ -332,10 +332,26 @@ def slots_available(self):
             return sys.maxsize
 
     @staticmethod
-    def validate_command(command: List[str]) -> None:
-        """Check if the command to execute is airflow command"""
+    def validate_airflow_tasks_run_command(command: List[str]) -> Tuple[Optional[str], Optional[str]]:
```

Comment:
Oh. This counts as a breaking change if anyone has written a custom executor and was calling `validate_command` (which they should be ideally) it would now blow up.

Given that maybe two separate functions might be the way to go?

Reply:
Yeah. I thought It might be an issue. I wiil just add back-compat method

Reply:
I was basically waiting if somone will raise it (as I thought I am overthinking it :)).